content/dogepedia/articles/becoming-a-dogecoin-developer.md

```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-How can shibes get involved with Dogecoin development?
+Dogecoin is an open-source project, meaning that anyone with some technical know-how can contribute to Dogecoin’s technical development. Even if you’re new to the space: it’s never too late to learn new tricks!  
+
+Dogecoin has an amazing team of developers and volunteers who are committed to welcoming new people into the space and helping them to learn and develop their skills. For example, core developer Ross Nicoll frequently runs streams via Twitch in which newcomers and old-timers alike can participate in programming tutorials and demonstrations. The community is extremely encouraging, too, and it’s very common to see old Shibes helping new Shibes as they enter the space.  
+
+Dogecoin has many projects through which you can help: Core, Libdogecoin, Gigawallet, Documentation, and websites to name but a few. [As emphasised] (https://twitter.com/tjstebbing/status/1480001128880230401?s=20) by the Dogecoin Foundation’s Timothy Stebbing: YOU can be a Dogecoin Dev just by picking-up your metaphorical shovel! Opensource communities are meritocracies: the more you do, the more you’re respected and the more you can be the change you want to see! 
+
+The Dogecoin developers are passionate about growing the space through which people can BUIDL the Dogecoin ecosystem. To this end, one of the key objectives behind [Libdogecoin] (https://github.com/dogecoinfoundation/libdogecoin) in creating bindings for many languages is making Dogecoin more accessible to a broader developer audience.

```

Comment:
instead of ')...in creating bindings...' perhaps change to ')...is to create bindings...'